community/bolt/LICENSES.txt

```diff
@@ -15,6 +15,7 @@ Apache Software License, Version 2.0
   Lucene Memory
   Lucene QueryParsers
   Netty/All-in-One
+  Objenesis
```

Comment:
do you know where Objenesis comes from now? I remember we killed it some time ago. Do we really need it back or we can exclude it?